src/cpp/qzmq/src/qzmqsocket.cpp

```diff
@@ -601,7 +601,6 @@ class Socket::Private : public QObject
 		}
 	}
 
-public slots:
 	void sn_read_activated()
```

Comment:
I'm surprised this didn't need an int arg?

Reply:
this signal is not boostified yet. i need to revert this line